english/polar-bear/polar-bear.md

```diff
@@ -25,6 +25,9 @@ the walruses quickly help the situation by using their tusks to pierce any
 marshmellows and create a rip in the fabric of reality.. you jump on the 
 walruses back..
 
+While riding the walrus you quickly pull the pin from the hand grenade you’ve been clenching and toss it in the general direct of the polar bear. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.